src/WebSocketServer/WebSocketUtility.cs

```diff
@@ -16,5 +17,12 @@ public static UInt160 TryParseScriptHash(string addressOrScriptHash, byte addres
 
             return scriptHash ?? UInt160.Zero;
         }
+
+        public static BigInteger TryParseBigInteger(string value)
```

Comment:
Why having this instead of using tryparse directly?

Reply:
I dont have the `3` line of code over and over again. Code redundancy is bad.